### DISCUSSION TITLE
Load settings to new releases?

### DISCUSSION THREAD

**Discussion by d-nx** [Jyers/Marlin#654] [Q&A] [answered]
I'm running 1.3.0 but would love to stay up to date with newer releases.

I've backed up my EEPROM settings as a .json file and was wondering if I could flash new releases but load my saved settings moving fwd.

I'm kind of new at this so any help is greatly appreciated.
Thanks!

**Comment by Cal-LF** [comment]
This may be in the works to save settings to the SD card.

**Reply by d-nx** [reply]
Good to know!

Just to confirm, if I were to go from 1.3.0 to 1.3.1 I would need to re-calibrate everything? or can I manually input each value from what I've got written down? (E-steps, probe offsets, z-offset, etc)

**Reply by darco415** [reply]
Currently your settings are saved to eeprom....reflashing a new fw will wipe out those settings. Compiling your own fw and saving to your local branch (desktop) is how most do it.

**Comment by Jyers** [comment, marked as answer]
You have a few options in this regard. I believe the output from M503 can be put into a file and run from the print menu. This should restore the previous settings. If you use octoprint you can also try this https://plugins.octoprint.org/plugins/eeprom_marlin/

**Reply by d-nx** [reply]
Thanks!

I successfully updated from 1.3.0 to 1.3.2 and all my settings carried over. I backed up my settings using the eeprom plugin in octoprint, but didn't have to load them. I wonder if the settings are saved in my SD card then read by the updated firmware?

Even though I'm not sure how it worked, I'm happy I was able to update and keep my settings.

PS
I'm loving the new menu ui color settings.
![ender screen](https://user-images.githubusercontent.com/80003983/116832284-6ac00780-ab82-11eb-9653-dfa1f1755475.jpeg)

**Comment by Tuncay-Ayhan** [comment]
> You have a few options in this regard. I believe the output from M503 can be put into a file and run from the print menu. This should restore the previous settings. If you use octoprint you can also try this https://plugins.octoprint.org/plugins/eeprom_marlin/

I've been successfully using the Octoprint plugin. It works great.
